server/src/api/pending-sentences/handler/add-pending-sentence-vote-handler.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response } from 'express'
+import { pipe } from 'fp-ts/lib/function'
+import { task as T, taskEither as TE } from 'fp-ts'
+import { StatusCodes } from 'http-status-codes'
+import { addPendingSentenceVoteCommandHandler } from '../../../application/pending-sentences/use-case/command-handler/add-pending-sentence-vote-command-handler'
+import { AddPendingSentenceVoteCommand } from '../../../application/pending-sentences/use-case/command-handler/command/add-pending-sentence-vote-command'
+import { createPresentableError } from '../../../application/helper/error-helper'
+
+export default async (req: Request, res: Response) => {
+  const command: AddPendingSentenceVoteCommand = {
+    clientId: req.client_id || null,
```

Comment:
QUESTION: Should it be possible for users that are not logged in to vote on sentences? I think only logged-in users should be able to review sentences because on the frontend we are hiding the Review option for users that are not logged in